keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/README.md

```diff
@@ -235,51 +452,101 @@ _This section must be completed when targeting beta graduation to a release._
+deleted nodes not being removed, or components failing to start due to missing
+taint tolerations, these could all be possible signs that a CCM rollout has
+failed.
+
+In addition there are several metrics exposed by the cloud-provider framework
+that is maintained by SIG Cloud Provider. If the external CCM is using the
+cloud-provider framework, the following metrics could inform about failing
+behavior, users should be observant of these metric values increasing over
+a short period of time, or after upgrade:
+
+* `cloud_provider_taint_removal_delay_seconds`
+* `initial_node_sync_delay_seconds`
+* `nodesync_error_total`
+* `nodesync_latency_seconds`
+* `update_loadbalancer_host_latency_seconds`
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
 
-  TBD for beta.
+  * Yes, tested manually by various providers, notably AWS and GCE.
 
 * **Is the rollout accompanied by any deprecations and/or removals of features, APIs,
 fields of API types, flags, etc.?**
 
-  TBD for beta.
+  * The provider options (e.g. `aws`, `azure`, etc), with the exception of `external`,
+    for the `--cloud-provider` flag of `kubelet`, `kube-apiserver`, and
+    `kube-controller-manager` are deprecated with the beta release.
+  * The in-tree cloud controller loops for `kubelet`, `kube-apiserver`, and
+    `kube-controller-manager` are deprecated with the beta release.
 
 ### Monitoring Requirements
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can an operator determine if the feature is in use by workloads?**
 
-  TBD for beta.
+  * In genreal, if pods are being scheduled to the cluster then this feature is
+    working as intended, and is in use by workloads.
+  * Workloads that explicitly use load balancer type services will depend on external CCMs
+    on providers that support the load balancer service controller.
+  * Workloads that depend on well-known zone and region labels on nodes are
+    also considered to be using this feature transitively.
 
 * **What are the SLIs (Service Level Indicators) an operator can use to determine
 the health of the service?**
 
-  TBD for beta.
+  * Pods scheduling, if the CCM node controller is not working properly the
+    rate of pod scheduling will decrease.
+  * Deleted node removal, if the CCM node lifecycle controller is not working
+    properly then terminated nodes will not be removed from the cluster.
+  * Load balancer services not being created or destroyed, if the CCM service
+    controller is not working properly, load balancer services will not be
+    created or removed.
 
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
 
-  TBD for beta.
+  * Nodes being initialized properly, zone and region labels being added.
+  * Pods being scheduled.
+  * Load balancer type services being created and destroyed.
+  * It is difficult to be more proscriptive about the SLOs of individual CCMs
```

Comment:
So let's change to something:

"The existing SLOs for functionalities X, Y and Z should be used - the change is supposed to be no-op from end-user perspective."

Reply:
ack, will update.